app/views/layouts/application.html.erb

```diff
@@ -8,13 +8,21 @@
         <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
         <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
         <%= render 'layouts/shim' %>
-        <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet">
+        <script src="https://kit.fontawesome.com/78a530eb20.js"></script>
```

Comment:
No need to add fontawesome link to the head if fontawesome is already added in the Gemfile